data/embeddings-creation.ts

```diff
@@ -33,10 +34,8 @@ await writeFile(
 function initProgressBar(): SingleBar {
   const progressBar = new SingleBar({}, cliProgress.Presets.shades_classic);
   const chunksCount =
-    SpChunks.length +
-    ConfluenceChunks.length +
-    PeopleChunks.length +
-    MiroChunks.length;
+  
+    GitHubChunks.length;
```

Comment:
hast wahrscheinlich aus Versehen überschrieben, hier der fix für den total count
```suggestion
    SpChunks.length +
    ConfluenceChunks.length +
    PeopleChunks.length +
    MiroChunks.length + 
    GitHubChunks.length;
```